Source/santactl/Commands/SNTCommandFileInfo.m

```diff
@@ -383,7 +383,6 @@ - (SNTAttributeBlock)rule {
     dispatch_once(&token, ^{
       [cmd.daemonConn resume];
     });
-    __block SNTEventState state;
     dispatch_semaphore_t sema = dispatch_semaphore_create(0);
     NSError *err;
```

Comment:
If there is a cs error, we shouldn't bother getting attributes from csc.

Reply:
Done, sort of